okapi-core/src/main/raml/okapi.raml

```diff
@@ -1128,6 +1128,31 @@ types:
         body:
           text/plain:
 
+/_/proxy/import/modules:
```

Comment:
Can you move it just before `/_/proxy/modules:` to have these very similar endpoints next to each other?

Reply:
ok